ivy/functional/backends/paddle/elementwise.py

```diff
@@ -153,95 +162,55 @@ def bitwise_and(
     return paddle.bitwise_and(x1, x2)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "complex")},
+    backend_version,
+)
 def ceil(x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None) -> paddle.Tensor:
-    x_dtype = x.dtype
-    if x_dtype in [
-        paddle.int8,
-        paddle.int16,
-        paddle.int32,
-        paddle.uint8,
-        paddle.float16,
-        paddle.complex64,
-        paddle.complex128,
-        paddle.bool,
-    ]:
-        if paddle.is_complex(x):
-            return paddle.complex(paddle.ceil(x.real()), paddle.ceil(x.imag()))
-        return paddle.ceil(x.astype("float32")).astype(x_dtype)
-    elif x_dtype == paddle.int64:
-        return paddle.ceil(x.astype("float64")).astype(x_dtype)
+    if paddle.is_complex(x):
+        return paddle.complex(paddle.ceil(x.real()), paddle.ceil(x.imag()))
     return paddle.ceil(x)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float16", "float32", "float64", "complex")},
+    backend_version,
+)
 def floor(x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None) -> paddle.Tensor:
-    x_dtype = x.dtype
-    if x_dtype in [
-        paddle.int8,
-        paddle.int16,
-        paddle.int32,
-        paddle.uint8,
-        paddle.float16,
-        paddle.complex64,
-        paddle.complex128,
-        paddle.bool,
-    ]:
-        if paddle.is_complex(x):
-            return paddle.complex(paddle.floor(x.real()), paddle.floor(x.imag()))
-        return paddle.floor(x.astype("float32")).astype(x_dtype)
-    elif x_dtype == paddle.int64:
-        return paddle.floor(x.astype("float64")).astype(x_dtype)
+    if paddle.is_complex(x):
+        return paddle.complex(paddle.floor(x.real()), paddle.floor(x.imag()))
     return paddle.floor(x)
 
 
-@with_unsupported_device_and_dtypes(
-    {"2.5.1 and below": {"cpu": ("bool", "bfloat16")}},
+@with_supported_dtypes(
+    {
+        "2.5.1 and below": (
```

Comment:
we could probably format these inputs to a single line like before as there's only 2 dtypes specified. Same applies to other changes where the length of the single line statement is less than `88` characters

Reply:
understood